### ISSUE TITLE
Scope stop in standalone mode doesn't refresh UI

### ISSUE THREAD

**Issue by abuehrle** [weaveworks/scope#2219]
env: docker for mac (1.13.0 (15072))
browser: chrome

```scope stop```  not removing UI from browser.  Can still click on objects with the app trying to find it. 

Although the agent is no longer running when you do a: ```docker ps```

**Comment by 2opremio** [comment]
@abuehrle That's just how the UI works. If the backend dies the UI is still alive trying to contact it (you will see an icon in the bottom left stating it's trying to connect).

**Comment by 2opremio** [comment]
@abuehrle Where you expecting a different behavior? Which one?

**Comment by abuehrle** [comment]
I was expecting a blank screen, since the probes are no longer present.  Otherwise it looks broken.  Maybe you guys could add a little beacon indicating that the probes are there/not there, like you have in Cloud.

**Comment by ekimekim** [comment]
The problem is that the app is also gone - there's no more website left to display that info.

**Comment by 2opremio** [comment]
> Otherwise it looks broken

What do you see? Can you post a screenshot?

**Comment by abuehrle** [comment]
I did happen to take a screen shot. On the details panel, you get the spinner: 

![screen shot 2017-02-08 at 4 37 44 pm](https://cloud.githubusercontent.com/assets/13420383/22760654/08fbff62-ee1d-11e6-95b1-27094e7f42ce.png)

**Comment by 2opremio** [comment]
Uhm, the topology keeps being rendered.

@abuehrle For how long? It's OK to cache ti for a while (a few seconds) but we may be caching it for too long.

**Comment by 2opremio** [comment]
I can reproduce, the _trying to reconnect_ icon is "breathing" but may not be clear enough.

<img width="2028" alt="screen shot 2017-02-09 at 01 06 26" src="https://cloud.githubusercontent.com/assets/2362916/22763310/45cd6984-ee64-11e6-9a28-235b0fa8b2da.png">


Maybe we could do something more explicit, like graying out the view on disconnections. @davkal / @bowenli Thoughts?

**Comment by abuehrle** [comment]
It may be caching for a very long time if you remain in the same view. Once you click on a different view, the nodes do disappear. Here's a recording: 

http://recordit.co/qlSMEk2UHn

**Comment by abuehrle** [comment]
I made a recording here: 
http://recordit.co/qlSMEk2UHn

It looks like if you stay in one view, it may be a very long time, but if you switch contexts then it seems fine.

**Comment by bowenli** [comment]
- Killing the server means the client might have indeterminate behavior because it's never expected to run in that state. Specifically, I think any end user that is stopping scope should understand that means the web page has nothing to connect to anymore.
- We don't have any way to determine if the probes are killed or just slow. I think keeping the elements around in case there is a network delay or something is a better experience than having them flash in and out every time we miss a connection by a second.

**Comment by abuehrle** [comment]
Understood @bowenii and I'm not sure it's such a big deal.  It might be good to mirror what Cloud does at some point, i.e. right away you can see that the probes are no longer connected: 

<img width="721" alt="screen shot 2017-02-08 at 9 06 09 pm" src="https://cloud.githubusercontent.com/assets/13420383/22767642/773ccfcc-ee42-11e6-9cc1-89cdb22a7f20.png">

**Comment by rade** [comment]
> I think keeping the elements around in case there is a network delay or something is a better experience than having them flash in and out every time we miss a connection by a second.

Yes, but what would be wrong with greying out the view after, say, 15s of disconnection?

**Comment by davkal** [comment]
IMO, short connection drops or scope restarts should not majorly change the UI. But if the backend is not coming online after a while (needs to be longer than scope restart times), I think it's a good idea to make the user aware as @rade suggested. The question is, do users still get value from looking at the last known state, in which case I would not grey it out, but rather make the disconnect message more prominent (and add a prominent back-online message when the backend is connected again).

**Comment by 2opremio** [comment]
https://github.com/weaveworks/scope/issues/2246#issuecomment-281993475 supports that the disconnected state in the UI is not clear enough.
